### PR TITLE
parity: 1.10.8 -> 1.11.8; parity-beta: init at 2.0.1

### PR DESCRIPTION
**Pull request by akru**
###### Motivation for this change

* https://github.com/paritytech/parity-ethereum/releases/tag/v2.0.0
* https://github.com/paritytech/parity-ethereum/releases/tag/v1.11.7

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Review by General-Beck — APPROVED** [review]
cmake 3.1+

File: pkgs/applications/altcoins/default.nix [hunk 1]
```diff
@@ -86,6 +86,7 @@ rec {
   };
 
   parity = callPackage ./parity { };
+  parity-beta = callPackage ./parity/beta.nix { };
```
[the remark this answers is not in the document]
Comment (akru) [reply pkgs/applications/altcoins/default.nix]: Yes, I'll update beta according to paritytech release policy in the same time as stable release.

[the remark this answers is not in the document]
Comment (General-Beck) [reply pkgs/applications/altcoins/default.nix]: LGFM

**Comment by xeji** [comment]
@GrahamcOfBorg build parity parity-beta

**Comment by xeji** [comment]
parity-beta fails on aarch64, but it could be a temporary error:

```
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 11, kind: WouldBlock, message: "Resource temporarily unavailable" }', libcore/result.rs:945:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.
error: Error during translation/LLVM phase.
```

**Comment by xeji** [comment]
Let's merge anyway and see what happens on Hydra.

**Comment by nanexcool** [comment]
Getting an error applying the patch:
```
applying patch /nix/store/fy2rfsfhymrmxb1hmabg7s98y6px0xn0-vendored-sources-1.11.patch
patching file .cargo/config
patch unexpectedly ends in middle of line
patch: **** malformed patch at line 112:  

builder for '/nix/store/lshhg676rx4y6jp2zacrp5dh594jq9sh-parity-1.11.8.drv' failed with exit code 2
error: build of '/nix/store/lshhg676rx4y6jp2zacrp5dh594jq9sh-parity-1.11.8.drv' failed
```

**Pull request merged**